src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MainConsumerImpl.java

```diff
@@ -28,6 +29,9 @@
 
     private final SlackMessagingService slackMessagingService;
 
+    @Value("${SEND_SLACK_MESSAGES}")
+    private boolean doSendMessages;
```

Comment:
probably worth having "slack" in the variable name, as weve been using "message" as terminology for a kafka message